src/Datadog.Trace.ClrProfiler.Managed/ScopeFactory.cs

```diff
@@ -41,8 +41,9 @@ public static Scope CreateOutboundHttpScope(Tracer tracer, string httpMethod, Ur
 
                 if (parent != null &&
                     parent.Type == SpanTypes.Http &&
-                    parent.GetTag(Tags.HttpMethod).Equals(httpMethod, StringComparison.OrdinalIgnoreCase) &&
-                    parent.GetTag(Tags.HttpUrl).Equals(UriHelpers.CleanUri(requestUri, removeScheme: false, tryRemoveIds: false), StringComparison.OrdinalIgnoreCase))
+                    parent.GetTag(Tags.InstrumentationName) != null &&
+                    httpMethod.Equals(parent.GetTag(Tags.HttpMethod), StringComparison.OrdinalIgnoreCase) &&
```

Comment:
As long as we're being extra-vigilant about NREs, should we check that `httpMethod`  and `requestUri` are not null?

Reply:
Good point, What do you think the behavior should be if either of those are null? Should we return null in that case?

Reply:
I ended up keeping them allowed and just changed it to call the static `string.Equals` so there's no NRE there